code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -66,51 +68,109 @@
 	var/supernatural = 0
 	var/purge = 0
 
+

```

Comment:
I basically just took our code and got a few things from bay.
Should look over that if there is more that we should take.